tools/cli/README.md

```diff
@@ -1,41 +1,75 @@
-coherent-guic-cli is a tool that helps you create and develop components of the GameUIComponents suite.
+The coherent-guic-cli is a tool that helps you create and develop components from the GameUIComponents suite. It can also setup a development environment with live reload that will allow you to easily create a UI using the components.
 
 # Getting Started
 
 To install the cli open a console and type:
-`npm i -g coherent-guic-cli`
+
+```
+npm i -g coherent-guic-cli
+```
 
 This will install the cli as a global module and will make it available from everywhere.
 
+## Available Commands
+
+|Command   |Description   |Arguments   |Usage   |
+|---|---|---|---|
+|create-project               |Create a new project from template                 | `<template>` `<directory>`|`coherent-guic-cli create-project webpack ./my-folder`|
+|create | [name], [directory] |           Create a new component | `coherent-guic-cli create my-component ./my-folder`|
+|build |[--watch]        | start the server|`coherent-guic-cli build`|
+|start:demo| N/A|              Start a development server and host the demo|`coherent-guic-cli start:demo`|
+|build:demo| [--dev] |      Create a production bundle of the demo. |`coherent-guic-cli build:demo`|
+|--help| N\A |      show help. |`coherent-guic-cli --help`|
+
 ## Usage
 
 ### Setting up an Environment
 
-You can use the `coherent-guic-cli` to create a project that uses GameUIComponents. It will setup the required tools to build and debug the code. Run
+If you want to create a UI using the components - use the `create-project` command. It allows you to setup a project that uses [Webpack](https://webpack.js.org/) or [Vite](https://vitejs.dev/). Open a console and type:
+
 ```
-coherent-guic-cli create-project <template> <directory>
+coherent-guic-cli create-project webpack ./my-folder
 ```
 
-`<template>` can be either 'webpack' or 'vite'.
+or
+
+```
+coherent-guic-cli create-project vite ./my-folder
+```
+
+This will automatically install all dependencies. After the project is ready - navigate to `my-folder` and type
+
+```
+npm run dev
+```
+
+this will start a development server with live reload enabled for both project types - webpack and vite.
+
+`<template>` can be `webpack` or `vite`.
 
 Example:
+
 ```
 coherent-guic-cli create-project vite ./my-folder
 ```
 
 This will create a new project inside `./my-folder` and will install the dependencies. After that you can navigate to the folder and start a development server:
+
 ```
 cd ./my-folder
 npm run dev
 ```
 
 ### Creating a Component
-After the installation is complete open a console and run:
 
-`coherent-guic-cli create my-first-guic components`
+You can use the CLI to create and develop custom components like the [Checkbox](https://coherentlabs.github.io/GameUIComponents/en/examples/checkbox/). It will setup the required environment for the development of a custom component. Use the create command to create a component:

```

Comment:
```suggestion
You can use the CLI to create and develop custom components like the [Checkbox](https://coherentlabs.github.io/GameUIComponents/en/examples/checkbox/). It will setup the required environment for the development of a custom component. Use the `create` command to create a component:
```